site/styles/_meta-sub-item.scss

```diff
@@ -5,10 +5,17 @@
   grid-template-rows: auto minmax(0, 1fr);
   margin-bottom: 15px;
   padding-bottom: 15px;
+  transition: height 0.5s ease;
+  @include media-breakpoint-up(sm) {
+    height: auto !important;
```

Comment:
This can go in the `@include media-breakpoint-up(sm)` block just below.

Reply:
- fixed as part of 'separate div inside `<s72-show-hide-switcher>`' css refactor